src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ColorSupplier cs = new ColorSupplier();

```

Comment:
```suggestion
        ColorSupplier colorSupplier = new ColorSupplier();
```

cs - not informative variable name